ArchiSteamFarm/Json/Steam.cs

```diff
@@ -471,17 +471,26 @@ internal class NonZeroResponse {
 		[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
 		internal sealed class PrivacyResponse {
```

Comment:
Instead of unsealing it from `readonly`, implement custom constructor like done in `TradeOffer`. All fields should be readonly after their creation, unless absolutely crucial to edit after their creation/deserialization.

Reply:
This still stands.